scm/env_helper.go

```diff
@@ -33,3 +36,21 @@ func GetWebHookServer(t *testing.T) string {
 	}
 	return serverURL
 }
+
+// Creates organization based GitHub app client to use in tests. If organization name is empty,
+// uses the default test organization name
+func GetTestClient(t *testing.T, org string) *github.Client {
```

Comment:
Test helpers should be in `_test.go` files. You should also call `t.Helper()` on the first line in test helpers.